python/remote-control-center/packaged.yaml

```diff
@@ -0,0 +1,66 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: An AWS Serverless Specification template describing your function.
+Globals:
+  Function:
+    Timeout: 30
+    Runtime: python3.7
```

Comment:
any reason for not 3.8?

Reply:
at the time of dev, there was python3.7 wasn't supported.  Updated with 3.8